src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -29,6 +34,32 @@ const pizzaColumns = {
   specialInstructions: 'SPECIAL_INSTRUCTIONS',
 };
 
+/**
+ * oracleDB out bind params for getting the results of an INSERT
+ * or UPDATE query
+ *
+ * @const
+ * @type {object[]}
+ */
+const pizzaOutBindParams = _.reduce(pizzaProperties,
+  (outBindParams, properties, name) => {
+    const bindParam = {};
+    bindParam.type = properties.type === 'string' ? OracleDB.STRING : OracleDB.NUMBER;
+    bindParam.dir = OracleDB.BIND_OUT;
+    outBindParams[`${name}Out`] = bindParam;
+    return outBindParams;
+  }, { idOut: { type: OracleDB.NUMBER, dir: OracleDB.BIND_OUT } });
+
+/**
+ * Query for updating a pizza record
+ *
+ * @const
+ */
+const pizzaPostQuery = dedent`INSERT INTO PIZZAS (NAME, BAKE_TIME, OVEN_TEMP, SPECIAL_INSTRUCTIONS)
+    VALUES (:name, :bakeTime, :ovenTemp, :specialInstructions)
+  RETURNING ID, NAME, OVEN_TEMP, BAKE_TIME, SPECIAL_INSTRUCTIONS
+  INTO :idOut, :nameOut, :ovenTempOut, :bakeTimeOut, :specialInstructionsOut`;
```

Comment:
trying out writing queries by hand here instead of constructing them with code, especially since this one is totally static